iceoryx_utils/source/posix_wrapper/access_control.cpp

```diff
@@ -104,43 +104,45 @@ bool AccessController::addPermissionEntry(const Category f_category,
     {
     case Category::SPECIFIC_USER:
     {
+        bool result{false};
         if (f_name.empty())
         {
             std::cerr << "Error: specific users must have an explicit name." << std::endl;
-            return false;
+            result = false;
         }
 
         auto id = posix::PosixUser::getUserID(f_name);
         if (!id.has_value())
         {
-            return false;
+            result = false;
         }
         else
         {
-            return addPermissionEntry(f_category, f_permission, id.value());
+            result = addPermissionEntry(f_category, f_permission, id.value());
         }
 
-        break;
```

Comment:
similarly here, some tools complain if there is no break, even if there is a return. Do we want to ignore such tools? I'm all for it. We just need to be conciuous about the decission